### DISCUSSION TITLE
キーボール39の基盤の直し方

### DISCUSSION THREAD

**Discussion by sayasaya2525** [Yowkees/keyball#687] [Q&A]
キーボール39のLEDを外した際に15のプリント基盤が剥がれてしまいました
14もつかなくどうすればいいかわからないので教えて欲しいです![image](https://github.com/user-attachments/assets/db4f497f-b74d-4389-b00e-447a06ffbbcd)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

Keyball39のトラックボール有り側とお見受けしました。
剥がれたパターンを復活させるには、下記のオレンジ点どうしをエナメル線かゴム被覆導線で接続していただけると回路が正常にもどります。
![39RLEDfix](https://github.com/user-attachments/assets/be10c08a-eddf-4b9f-8755-41f4e5373588)
お手数ですがご対応お願いいたします。

**Reply by sayasaya2525** [reply]
ご回答ありがとうございます。
土日に購入して試してみます！
